app/upw/controllers/start.js

```diff
@@ -11,6 +35,47 @@ class StartUnpaidWork extends BaseController {
 
     super.locals(req, res, next)
   }
+
+  // eslint-disable-next-line consistent-return
```

Comment:
Is this a needed comment?

Reply:
It is - eslint doesn't really understand the structure of the way our classes are used by the framework